fabricTools/deployments/peer/fabric-peer-template-couchDB.yaml

```diff
@@ -36,15 +36,15 @@ spec:
       - name: fabric-tools-secrets
       containers:
       - name: couchdb{nodeNum}
-        image: hlfakstemplate.azurecr.io/hyperledger/fabric-couchdb:1.4.8
+        image: hyperledger/fabric-couchdb:0.4.21
         ports:
          - containerPort: 5984  
         volumeMounts:
          - mountPath: /opt/couchdb/data/
            name: fabric-storage
            subPath: peer{nodeNum}-couchdb
       - name: peer{nodeNum}
-        image: hlfakstemplate.azurecr.io/hyperledger/abs-fabric-peer:1.4.8
+        image: hyperledger/fabric-peer:1.4.8
```

Comment:
In case of peer deployments, we would also have to modify the chaincode images (which are currently point to our ACR), to point them to their Docker Hub counterparts:

```
hlfakstemplateoss.azurecr.io/hyperledger/abs-fabric-ccenv:1.4.8 TO  hyperledger/fabric-ccenv:1.4.8
hlfakstemplateoss.azurecr.io/hyperledger/abs-fabric-golangruntime:1.4.8 TO hyperledger/fabric-baseos:amd64-0.4.21
hlfakstemplateoss.azurecr.io/hyperledger/abs-fabric-nodejsruntime:1.4.8 TO hyperledger/fabric-baseimage:amd64-0.4.21
```

Reply:
Done